docs/animations/AnimationSet.md

```diff
@@ -63,50 +61,55 @@ You can change the way how the animation interpolates between keyframes by defin
 | Quintic    | Create an animation that accelerates or decelerates using the formula f(t) = t5                                                                         | ![QuinticEase](https://docs.microsoft.com/en-us/dotnet/framework/wpf/graphics-multimedia/media/quinticease-graph.png)     |
 | Sine       | Creates an animation that accelerates or decelerates using a sine formula                                                                               | ![SineEase](https://docs.microsoft.com/en-us/dotnet/framework/wpf/graphics-multimedia/media/sineease-graph.png)           |
 
-***Note:** EasingType is used only when AnimationSet.UseComposition == false*  
```

Comment:
Many of these docs changes seem unrelated to this PR. Can you separate these into their own PR?

Reply:
@Vijay-Nirmal is fixing a lot of these with #1748, and I'd love for him to review this to make sure it wouldn't conflict with his changes.

Reply:
I have changed a lot of things in #1748 to match the [.template.md](https://github.com/Microsoft/UWPCommunityToolkit/raw/master/docs/.template.md) and docs.microsoft.com format. For example, I changed `**Note:**` to `[!NOTE]`. @WilliamABradley What do you think?

> These changes I have made fixed the issues with the documentation pages that caused it not display properly in the MarkdownTextBlock, even with these changes

@WilliamABradley Actually, we can't fix the display issue because [docs.microsoft.com](https://docs.microsoft.com/en-us/contribute/ops-crr/openpublishing/docs/partnerdocs/gfm?toc=/contribute/TOC.json&bc=/contribute/breadcrumb/toc.json) uses [DocFX Flavored Markdown](http://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html). As @nmetulev mention [here](https://github.com/Microsoft/UWPCommunityToolkit/issues/1740#issuecomment-357139872), we have to make a lot of changes to MarkdownTextBlock parser and renderer to fix this issue.

Reply:
@Vijay-Nirmal I have had a look at your changes, it doesn't seem like it would be too hard to merge, Things for you to keep note of, is that currently the MarkdownTextBlock does not support Code Blocks inside of a List Block, in areas of the documentation I removed the Listing areas that used codeblocks, and used bold instead, and other things that didn't present nicely in the MarkdownTextBlock.

There were also several areas of the Documentation where the formatting were broken, like a space at the end ticks of a code block, meaning the rest of the page became the Code Block, as well as other formatting issues.

You are more than welcome to take a copy of the latest commit locally and investigate the changes if you wish (Note, currently the Sample app takes the GitHub master branch copies over the local copies, so you will have to use VS Code MD Preview or some other previewer).

Reply:
I don't think using bold in place of a list is a good idea.

Reply:
@Vijay-Nirmal This is the output:

![Output](https://i.imgur.com/FbfagcG.png)

It looks a lot better than before, when none of the Code Blocks would display properly.

Reply:
This is what it looks like in a list:
![Example](https://i.imgur.com/BuOydh5.png)

Reply:
This is the old look. Actually, I prefer the old format with the new look. Also, How the code block in the list is working properly when the new one doesn't?
![applicationframehost_2018-01-20_11-32-05](https://user-images.githubusercontent.com/24755596/35180401-bc261460-fdd5-11e7-9c69-92d24a99d23c.png)

Reply:
Hmm, that is strange indeed, I will take a look. 

I personally think that the list text should be more intense than the Sample Code/Sample Output text, that way it is more obvious that it is part of that list element.

Maybe bold list items, italic text?

Reply:
Also if you look at it, the formatting is wrong for the code blocks, also the Markdown didn't have any spaces, which is invalid Markdown anyway.

Reply:
We can change Sample Code/Sample Output text to italic or completely remove it.

> Also if you look at it, the formatting is wrong for the code blocks, also the Markdown didn't have any spaces, which is invalid Markdown anyway.

Yes, but it that an unfixable bug?

The reason we cant make the list as bold is that the line describes the example. Using bold in the description is not the good idea. A description can be even more than five lines.

Reply:
After long investigation, the reason the old look works, is because of ProcessDocs() in [Sample.cs](https://github.com/Microsoft/UWPCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp.SampleApp/Models/Sample.cs), which converted all Code Blocks into Quote Blocks.

I will investigate if there is a way I can replicate this structure easily for Code Blocks.